### PR TITLE
PLT-3820 Fix preview flickering on Edge and remove previews from DOM when not expanded

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Fix the flickering that sometimes occurs when posting image links on Edge. Also prevented previews from rendering when they weren't going to be displayed. This should hopefully also help with the issue of GIFs eating up CPU when collapsed.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3820

### CODE REVIEW

**Comment by enahum** [comment]
+1

File: webapp/components/post_view/components/post_body_additional_content.jsx [hunk 1]
```diff
@@ -73,8 +78,41 @@ export default class PostBodyAdditionalContent extends React.Component {
...
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    isLinkToggleable() {
+        const link = this.state.link;
+        if (!link) {
```
Comment (hmhealey) [inline webapp/components/post_view/components/post_body_additional_content.jsx]: There's some duplicated logic here. Otherwise, +1

Comment (jwilander) [inline webapp/components/post_view/components/post_body_additional_content.jsx]: Oh jeez, how did that happen. Fixing..

**Comment by jwilander** [comment]
@hmhealey fixed

**Comment by hmhealey** [comment]
+1 again then

**Pull request merged**